### PR TITLE
Minor changes on home page (introduced Tailwind CSS)

### PR DESCRIPTION
**Pull request by jayp822**
* uses tailwind CSS there is not any conflict with other people's CSS code. Make sure to npm i so tailwind can be in the package.json and be working*
- Made footer year dynamic
- changed Navbar to be responsive and made join us button cleaner
- made Roles carousel be automatic instead of on arrow press. 
- made changes to font and text size on home page to be responsive.
- in a smaller screen the text and image dont overlap instead it goes text, image, text, image in a vertical format.

### CODE REVIEW

**Comment by jayp822** [comment]
*did this before knowledge of the Figma board

**Comment by KelseySt** [comment]
Please include screenshots of changes so we get an idea of what to look through quickly!

**Comment by ggliv** [comment]
Not entirely sure why I was requested for review, I'm going to step aside on this one.

File: src/app/components/NavBar.css [hunk 1]
```diff
@@ -4,15 +4,15 @@ a {
...
+/* a:visited {
     text-decoration: none;
     color: white;
 }
 
 a:hover {
     text-decoration: underline;
     color: white;
-}
+} */
```
Comment (ggliv) [inline src/app/components/NavBar.css:15]: I will say that I think commenting things out like this (and [this](https://github.com/DevDogs-UGA/DevDogs-Website/pull/42/files#diff-976ddd751899c9f095b17db0bbdc8da3eec0728d8a25d6d1ee6ed45dbc7b4f6dR58-R60)) is bad practice - Git is keeping track of all your changes, so things like this aren't lost forever if you just delete them. Commenting them out just clutters the reading experience.

**Review by ggliv — CHANGES_REQUESTED** [review]

**Comment by jayp822** [comment]
Kelsey can you drop this pull request and checkout my new one? I changed some things after looking at the figma board

**Pull request closed without merging**